src/site/content/en/blog/redbus-inp/index.md

```diff
@@ -0,0 +1,141 @@
+subhead: |
+  INP optimizations helped redBus increase sales by ~7%
+description: |
+  INP optimizations helped redBus increase sales by ~7%
+authors:
+  - amitkumar
+  - saurabhrajpal
+date: 2023-05-10
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/3164So5aDk7vKTkhx9Vm.png
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/TR9V6jk32xwz5ACzj0RE.png
+alt: A composition of the redBus logo, at left, with the INP thresholds offset at right.
+tags:
+  - blog
+  - web-vitals
+  - case-study
+---
+
+The web and its capabilities are evolving at a fast pace. You can now build rich and full-featured experiences on the web that can achieve much of what native applications can in terms of capabilities.
+
+JavaScript is a primary driver for interactivity across the web, but if it's not used with care, interactions can feel sluggish, and lead users to perceive your website to be non-responsive or altogether broken. Thankfully, the [Interaction to Next Paint (INP)](/inp/) metric was created to address this specific user experience problem.
+
+INP measures all interactions made with a page during its lifecycle, and reports a single value that is representative of a website's speed in responding to user inputs. Simply put, if a page's INP is at or below the [good threshold](/inp/#what-is-a-good-inp-score), all interactions made with a page can be said to be reliably fast.
+
+[redBus](https://www.redbus.in/), a bus booking and ticketing website based in India, undertook a substantial effort to improve their website's INP, even during the time when it was still considered an experimental metric. As a result of their efforts, they were able to increase sales by 7%, illustrating once again the relationship between web performance and business health. Here's what redBus did to improve their website's INP.
+
+## Top goals
+
+When redBus set out to optimize their website's INP, they had three goals in mind:
+
+1. Provide a better user experience for users by focusing on interaction latency regardless of network speed and device capabilities.
+2. Optimize their website to keep interactions as fast as possible.
+3. Ensure that responses from their API were as light as possible to ensure fast data transfer to the front end.
+
+## The journey
+
+redBus categorized interaction latency in two ways:
+
+1. Interaction latency caused by blocking JavaScript on the client. When interactions use excessive JavaScript that blocks the main thread, rendering is delayed, and this affects a page's INP.
+2. Network latency caused by API calls. While network activity is not something that INP measures, an interaction reliant on a call to a remote API can feel sluggish on slower networks, or if requests result in large responses.
+
+redBus was initially quite confident that the INP for their website would be good, but the [Real User Monitoring (RUM)](https://en.wikipedia.org/wiki/Real_user_monitoring) data for this metric at the 95th percentile told a different story.
```

Comment:
They use web-vitals.js as their RUM but say they are looking at 95th percentile. Web-vitals.js uses 75th percentile so is this a typo?

Reply:
Ignore. Percentile is calced after collection. So they could use 95th percentile.